rpi5.yaml

```diff
@@ -0,0 +1,260 @@
+desc: "Xen for Raspberry 5 comprehended"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/zephyr-dom0-xt"
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_BUILD_DIR: "build"
+
+  DOMD_BUILD_DIR: "build/domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "image"
+
+  ZEPHYR_DOMD_MACHINE: "rpi_5_xen_domd"
+  ZEPHYR_DOMD_TARGET: "zephyr/samples/basic/blinky"
+  ZEPHYR_DOMD_DIR: "zephyr"
+  ZEPHYR_DOMD_BUILD_DIR: "build_d"
+
+  ZEPHYR_DOMU_MACHINE: "xenvm"
+  ZEPHYR_DOMU_TARGET: "zephyr/samples/synchronization"
+  ZEPHYR_DOMU_DIR: "zephyr"
+  ZEPHYR_DOMU_BUILD_DIR: "build_u"
+
+common_data:
+  # Sources used by domd
+  domd_sources: &DOMD_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: "scarthgap"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-common.git"
+      rev: "scarthgap"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-rpi5.git"
+      rev: "init_new"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-rpi5-prod-devel.git"
+      rev: "init_new"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-selinux"
+      rev: "master"
+
+  # Zephyr sources
+  zephyr_sources: &ZEPHYR_SOURCES
+    - type: west
+      url: "https://github.com/GrygiriiS/zephyr.git"
```

Comment:
This is not a good idea to merge private repo into xen-troops product.
This has to be merged into the xen-troops repo.

Reply:
I understand. This is temporary solution until that branch will be merged

Reply:
Sorry, but this PR should be in [WIP] and draft until developer's branches will be merged into xen-troops.

Reply:
Now it is [WIP]